app/models.py

```diff
@@ -0,0 +1,17 @@
+import typing as t
+
+from pydantic import BaseModel
+
+
+class AdminTable(BaseModel):
+    table_name: t.Optional[str] = None
+    visible_columns: t.Optional[list[str]] = None
+    visible_filters: t.Optional[list[str]] = None
+    rich_text_columns: t.Optional[str] = None
+    link_column: t.Optional[str] = None
+    menu_group: t.Optional[str] = None
```

Comment:
If we're using new style annotations here, can we do `str | None` instead? I can't remember if that works with Pydantic.

With Piccolo we support older versions of Python, so use `t.Optional`, but as this is a Docker container, we can target more recent Python versions (maybe even just 3.13 for now).

Reply:
Based on the [Pydantic docs](https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields) we need this `Not required, can be None, defaults to None` and that should be written as `Optional[str] = None` or `str | None = None`. I think in our case the model should be like this and 
```python
class AdminTable(BaseModel):
    table_name: str | None = None
    visible_columns: list[str] | None = None
    visible_filters: list[str] | None = None
    rich_text_columns: str | None = None
    link_column: str | None = None
    menu_group: str | None = None


class AdditionalConfig(BaseModel):
    tables: list[AdminTable] | None = None
    sidebar_links: dict[str, str] | None = None
```

that work both with values or None.

Reply:
OK, cool - keep it as `Optional` then.

Reply:
So then I should leave everything as it was?